lib/src/paginator/Paginator.tsx

```diff
@@ -155,6 +155,7 @@ const DxcPaginatorContainer = styled.div`
 `;
 
 const SelectContainer = styled.div`
+  min-width: 82px;
```

Comment:
I recommend removing the `max-width` to enable large numbers when there's available space and changing the `min-width` to 5.25rem in order to increase the value depending on the root font size.

Reply:
Done